### ISSUE TITLE
Suggestion: make the subscribe button more visible

### ISSUE THREAD

**Issue by eskatos** [apache/incubator-ponymail#101]
The subscribe button is down the list of messages.
Someone had to tell me it's there before I found it.
It should be more visible, maybe somewhere at the top.

**Comment by gmcdonald** [comment]
I don't agree it is hard to find, but it is floating a bit in the middle of nowhere. 

Perhaps having two buttons, exactly opposite the 'Show Next 10' buttons would be better, meaning a Subscribe link at the top and bottom left of the list of messages.
